core/server/app.js

```diff
@@ -22,12 +23,18 @@ module.exports = function setupParentApp() {
     // (X-Forwarded-Proto header will be checked, if present)
     parentApp.enable('trust proxy');
 
+    parentApp.use(responseTime());
```

Comment:
It's way more code then just replacing a single line.

```
var startTime = process.hrtime();
var endTime = process.hrtime(startTime);
> [ 0, 11835251 ]
var time = endTime[0] * 1e3 + endTime[1] * 1e-6
> 9.506681
time.toFixed(3) + 'ms'
> 9.50ms
```

I would like to avoid copying their logic over to Ghost. What do you think?